### ISSUE TITLE
[Feature Request]: add Swagger - automated API documentation

### ISSUE THREAD

**Issue by bugITwhisperer** [danielschuster-muc/potter-db#810]
### Project

API (api.potterdb.com)

### Describe the feature you want to request

Self-explanatory Swagger documentation for both GraphQL and REST API:
- https://docs.potterdb.com/apis/rest 
- https://docs.potterdb.com/apis/graphql

**Comment by danielschuster-muc** [comment]
We already have our Swagger / OpenApi Specification, have a look at the [endpoint](https://api.potterdb.com/v1/openapi) or with rswag autogenerated [file](https://github.com/danielschuster-muc/potter-db/tree/master/backend/app/documentation/v1). 
However it is doesn't have graphql implemented yet. 
Regarding displaying the interactive swagger ui, I would prefer to add it into the [docs](https://docs.potterdb.com). 

I am looking forward to hearing any ideas and feedback regarding the whole OAS/Swagger topic.

**Comment by bugITwhisperer** [comment]
@danielschuster-muc : Yeah, my idea was to have the interactive swagger ui displayed in the documentation - sorry for not being crystal clear here :)

**Comment by danielschuster-muc** [comment]
@bugITwhisperer sounds good! Feel free to create a pr for adding the swagger ui to the docs.

**Comment by manoellvitor** [comment]
@danielschuster-muc , There are some OAS disconformity: 

- Error:
Instance failed to match all required schemas (matched only 0 out of 2) --- Object instance has properties which are not allowed by the schema: ["data";"links";"meta"]
![image](https://github.com/danielschuster-muc/potter-db/assets/20465169/e44b5443-2a5b-40bf-a091-a67279d7229f)

```JSON

{
    "meta": {
        "pagination": {
            "current": 80,
            "first": 1,
            "prev": 79,
            "records": 7
        },
        "copyright": "Copyright © Potter DB 2024",
        "generated_at": "2024-03-06T15:46:56.357+00:00"
    },
    "links": {
        "self": "https://api.potterdb.com/v1/books?page%5Bnumber%5D=80&sort=worryingly",
        "current": "https://api.potterdb.com/v1/books?page[number]=80&sort=worryingly",
        "first": "https://api.potterdb.com/v1/books?page[number]=1&sort=worryingly",
        "prev": "https://api.potterdb.com/v1/books?page[number]=79&sort=worryingly"
    },
    "data": [
        
    ]
}

```

![image](https://github.com/danielschuster-muc/potter-db/assets/20465169/35142674-8e43-4c23-82d1-363e13666933)

**Comment by danielschuster-muc** [comment]
> @danielschuster-muc , There are some OAS disconformity:
> 
> * Error:
>   Instance failed to match all required schemas (matched only 0 out of 2) --- Object instance has properties which are not allowed by the schema: ["data";"links";"meta"]
>   ![image](https://private-user-images.githubusercontent.com/20465169/310564527-e44b5443-2a5b-40bf-a091-a67279d7229f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIyNzY1NDcsIm5iZiI6MTcxMjI3NjI0NywicGF0aCI6Ii8yMDQ2NTE2OS8zMTA1NjQ1MjctZTQ0YjU0NDMtMmE1Yi00MGJmLWEwOTEtYTY3Mjc5ZDcyMjlmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA1VDAwMTcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1NWI0MzdhOWJiNzlkMzFhMzc4MDU1MTMyMDQyNTNmYTkzNjJlMGU2NmJjOGQ0NWE3N2Y1N2ViMTRhZjgyMGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9jz5-phQXNk0YIQizZgCg2efWEXFm_8bvR5MROroJTk)
> 
> ```json
> {
>     "meta": {
>         "pagination": {
>             "current": 80,
>             "first": 1,
>             "prev": 79,
>             "records": 7
>         },
>         "copyright": "Copyright © Potter DB 2024",
>         "generated_at": "2024-03-06T15:46:56.357+00:00"
>     },
>     "links": {
>         "self": "https://api.potterdb.com/v1/books?page%5Bnumber%5D=80&sort=worryingly",
>         "current": "https://api.potterdb.com/v1/books?page[number]=80&sort=worryingly",
>         "first": "https://api.potterdb.com/v1/books?page[number]=1&sort=worryingly",
>         "prev": "https://api.potterdb.com/v1/books?page[number]=79&sort=worryingly"
>     },
>     "data": [
>         
>     ]
> }
> ```
> 
> ![image](https://private-user-images.githubusercontent.com/20465169/310565041-35142674-8e43-4c23-82d1-363e13666933.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIyNzY1NDcsIm5iZiI6MTcxMjI3NjI0NywicGF0aCI6Ii8yMDQ2NTE2OS8zMTA1NjUwNDEtMzUxNDI2NzQtOGU0My00YzIzLTgyZDEtMzYzZTEzNjY2OTMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA1VDAwMTcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZDljMDBiNTdlOTQzMTlmNmRhZWJjMGYyOTA2ODg4MTUzZWUxZDZmNTJkYzA2Y2IxMzIxMTdlY2ZlNTZiMzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7dBDt_eyO5RNnIGKcv5-nk9t4cb3sDvO15c9GWTj_ME)

Sorry for the late reply. Could you open a seperate issue for this problem, if it still occurs?
